bot/inLineButtons_quest.py

```diff
@@ -116,8 +116,31 @@ class TestStates(StatesGroup):
 
 user_answers = {}
 
+async def set_main_menu(bot:Bot):
+    main_menu_commands = [
+        BotCommand(command='/start',description='Ah Shit,here we go again.'),
+        BotCommand(command='/help',description= 'Справка по работе бота'),
+        BotCommand(command='/about_us',description='Команда, которая расскажет тебе про наши отделы'),
+        BotCommand(command='/quest',description= 'Пройди квест для определения своего отдела'),
+    ]
+    await bot.set_my_commands(main_menu_commands)
+dp.startup.register(set_main_menu)
+
+
+async def quest_menu(bot: Bot):
+    main_menu_commands = [
+        BotCommand(command='/back', description='Возвращает в начало квеста'),
+        BotCommand(command='/exit', description='Выходит из состояния квеста')
+    ]
+    await bot.set_my_commands(main_menu_commands)
```

Comment:
Is it being used? I thought we decided not to use "special" menu for `/quest` as it is impossible to refresh menu

Reply:
I was able to implement two menus in our code

Reply:
Okay. I tested it. On desktop version for MacOS. It doesn't change menu **immediately**, but after quitting chat and entering again it works
<img width="620" alt="image" src="https://github.com/user-attachments/assets/d5cb7eaf-0075-4840-af3e-e8b16e98722f">
<img width="620" alt="image" src="https://github.com/user-attachments/assets/74086184-a6f0-4549-b8d7-099a99cfe540">

Reply:
> I was able to implement two menus in our code

Yeah. I see it now. It works not super correctly. Buuuuuut I think it is because of the Telegram working process